cmd/server/server.go

```diff
@@ -122,6 +124,12 @@ func main() {
 	r.Get("/login", handleLogin)
 	r.Get("/login/authorized", handleAuth)
 
+	if !*dev { // static
+		workDir, _ := os.Getwd()
+		filesDir := http.Dir(filepath.Join(workDir, "frontend", "dist"))
+		FileServer(r, "/static", filesDir)
```

Comment:
This does not play nice with the fronted, as it tries to fetch JSON API under `/static/json/*`

Reply:
This is worked around when building a frontend with `npm run build -- --env baseUrl=/json`